app/models/claim/litigator_hardship_claim.rb

```diff
@@ -2,8 +2,9 @@ module Claim
   class LitigatorHardshipClaim < BaseClaim
     route_key_name 'litigators_hardship_claim'
 
-    validates_with ::Claim::LitigatorHardshipClaimValidator
-    validates_with ::Claim::LitigatorSupplierNumberValidator
+    validates_with ::Claim::LitigatorHardshipClaimValidator,
+                   unless: proc { |c| c.disable_for_state_transition.eql?(:all) }
+    validates_with ::Claim::LitigatorSupplierNumberValidator, if: proc { |c| c.draft? }
```

Comment:
I guess that this validator isn't relevant for this work but it is worth making consistent with the `Claim::LitigatorClaim` class.

Reply:
Yes, I added this for consistency. I can't see any reason why we should be validating supplier numbers differently on hardship or interim claims so it made sense to bring them in line.

I think they should all the litigator claim models should be consistent now (unless I'm missing something obvious, which is quite possible 😀 )